source/bossbar.rst

```diff
@@ -46,18 +45,18 @@ Changing an active Boss Bar
 Boss bars are mutable and listen for changes on their object,
 the in-game view will change automatically without having to manually refresh it!
 
-Therefore, if this boss bar is currently active
+Therefore, if this boss bar is currently active...
```

Comment:
why?

Reply:
Represents the thought "trailing off" into code, as opposed to simply cutting off mid-sentence. Less jarring to the reader. Open to using em/en dashes instead.